src/Algebra/Structures/Pointed.agda

```diff
@@ -0,0 +1,49 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Defines `IsPointedMonoid`
+-- intermediate between `Monoid` and `SemiringWithoutAnnihilatingZero`
+--
+-- By contrast with the rest of `Algebra.Structures`, this is modelled
+-- on an underlying `RawMonoid`, rather than a 'flattened' such signature
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Algebra.Bundles.Raw
+import Algebra.Structures as Structures
+open import Data.Nat.Base as ℕ using (ℕ)
+open import Data.Unit.Base
+open import Level using (Level; _⊔_; Lift)
+
+import Algebra.Definitions.RawMonoid as Definitions
+
+module Algebra.Structures.Pointed where
+
+private
+  variable
+    c ℓ : Level
+
+
+------------------------------------------------------------------------
+-- Structures with 1 binary operation & 2 elements
+------------------------------------------------------------------------
+
+record IsPointedMonoid
+  (rawMonoid : RawMonoid c ℓ)
+  (• : RawMonoid.Carrier rawMonoid) : Set (c ⊔ ℓ)
```

Comment:
I'm confused. You don't have a not equal to assumption between `•` and `ε` so there's not necessarily two elements...

Reply:
Absolutely! Degenerate instances are/should always be possible: we don't insist that `0#` not be equal to `1#` in `(Semi)ring*`, for example.
So perhaps the comment is bad, but this was mindless/lazy cut-and-paste from `Algebra.*`...